com.eviware.soapui.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: com.eviware.soapui
+
+# Using 19.08 since currently requires Java 16
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+
+command: soapui-launcher.sh
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-documents
+  - --persist=.soapuios
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk/install.sh
+
+  - name: SoapUI
+    buildsystem: simple
+    build-commands:
+      - mv bin lib soapui-settings.xml wsi-test-tools /app
+      - install -Dpm755 -t /app/bin soapui-launcher.sh
+      - install -Dpm644 -t /app/share/icons/hicolor/256x256/apps com.eviware.soapui.png
+      - install -Dpm644 -t /app/share/applications com.eviware.soapui.desktop
+      - install -Dpm644 -t /app/share/metainfo com.eviware.soapui.metainfo.xml
+    cleanup:
+      - /bin/*.log
+      - /bin/actions
+      - /bin/ext
+      - /bin/listeners
+    sources:
+      - type: archive
+        url: https://dl.eviware.com/soapuios/5.7.0/SoapUI-5.7.0-linux-bin.tar.gz
```

Comment:
just to confirm: this only contain pure JAVA and architecture specific binarie, doesn't it?

Reply:
Correct, I only see jar files and script files in the tarball.
I haven't perform a deep dive, but it doesn't seam to use any JNI to restrict it to any architecture.